### PR TITLE
76818: Add jobs for reading S3 csv and sending nod notification emails

### PR DESCRIPTION
**Pull request by dfong-adh**
## Summary

- *This work is behind a feature toggle (flipper): YES/NO*
No

This PR adds two jobs - one to load a list of email addresses from a CSV stored in S3, and another job to send emails via the VANotify service. This is owned by the Benefits Decision Reviews team.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/76818

## Testing done

- [x] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
No old behavior - these are new jobs
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
This has been tested locally and will be verified in staging.
- *What is the testing plan for rolling out the feature?*
 The DecisionReview::NodEmailLoaderJob will only be triggered manually as needed.

## What areas of the site does it impact?
This will impact the sidekiq job queues as this job will use the same queues. This feature also loads a single CSV file from S3.

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: app/sidekiq/decision_review/nod_send_email_job.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    sidekiq_options retry: false
+
+    LOG_PARAMS = {
+      key: :nod_send_email_job,
+      form_id: '10182',
+      user_uuid: nil
+    }.freeze
+
+    def perform(email, template_id, line)
+      Rails.logger.debug { "Starting send email job line: #{line} #{email}" }
```
Comment (data-doge) [inline app/sidekiq/decision_review/nod_send_email_job.rb:19]: Talked on call - will remove

File: app/sidekiq/decision_review/nod_email_loader_job.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    LOG_PARAMS = {
+      key: :nod_email_loader_job,
+      form_id: '10182',
+      user_uuid: nil
+    }.freeze
+
+    def perform(file_name, template_id, s3_config = Settings.decision_review.s3)
+      emails = get_emails(file_name, s3_config)
+
+      line = 1
```
Comment (dfong-adh) [inline app/sidekiq/decision_review/nod_email_loader_job.rb:21]: Skip header in CSV while processing emails

**Review by anniebtran — COMMENTED** [review]
looks great! happy to fully approve once you remove the email from the log

**Review by data-doge — DISMISSED** [review]
Thanks @dfong-adh, LGTM

**Review by anniebtran — APPROVED** [review]
LGTM

**Review by ryan-mcneil — APPROVED** [review]
👍

**Pull request merged**